packages/messages/src/browser/style/notifications.css

```diff
@@ -192,17 +192,11 @@
 }
 
 .theia-notification-actions > .expand {
-    transform: rotate(0deg) scale(1.5);
-}
-.theia-notification-actions > .expand:active {
-    transform: rotate(0deg) scale(1.8);
+    transform: rotate(180deg);
 }
 
 .theia-notification-actions > .collapse {
-    transform: rotate(-90deg) scale(1.5);
-}
-.theia-notification-actions > .collapse:active {
-    transform: rotate(-90deg) scale(1.8);
+    transform: rotate(0deg);
 }
```

Comment:
Minor: We don't need that rule anymore (rotating by 0deg does nothing) .

Reply:
:man_facepalming: I removed the rule, thanks!